aftership-prestashop-14x/aftership/v1/index.php

```diff
@@ -128,6 +135,14 @@ function orders() {
 	$last_created_at = gmdate('Y-m-d H:i:s', $last_created_at);
 	$last_updated_at = gmdate('Y-m-d H:i:s', $last_updated_at);
 
+	// by default handle as old version
+	$time_criteria = "AND o.`date_add` > '".$last_created_at."' AND o.`date_upd` > '".$last_updated_at."'";
+
+	// handle as new version if created_at_max and updated_at_max are set
+	if ($created_at_max != NULL && $updated_at_max != NULL) {
```

Comment:
I need both created_at_max and updated_at_max to be optional
we won't use created_at_max, just reserve for future use